From 478a9ee95e8e57c9ab8c9fbce4e2aa768bc68577 Mon Sep 17 00:00:00 2001
From: Dominik Poggel <pog@iesy.com>
Date: Mon, 2 Jan 2023 16:12:00 +0100
Subject: [PATCH 5/5] arm64: imx8mm: add mem-parameter to boot-cmd

---
 include/configs/imx8mm_evk.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/configs/imx8mm_evk.h b/include/configs/imx8mm_evk.h
index 5c19f283dd..0c9f8a37ae 100644
--- a/include/configs/imx8mm_evk.h
+++ b/include/configs/imx8mm_evk.h
@@ -117,6 +117,7 @@
 	CONFIG_MFG_ENV_SETTINGS \
 	JAILHOUSE_ENV \
 	"script=boot.scr\0" \
+	"memory=1024MB\0" \
 	"image=Image\0" \
 	"splashimage=0x50000000\0" \
 	"console=ttymxc2,115200\0" \
@@ -182,7 +183,7 @@
 */
 
 #define CONFIG_BOOTCOMMAND \
-	"setenv bootargs console=${console} root=${mmcroot} earlycon=ec_imx6q,0x30880000,115200; " \
+	"setenv bootargs console=${console} root=${mmcroot} earlycon=ec_imx6q,0x30880000,115200 mem=${memory};" \
 	"fatload mmc ${mmcdev} ${loadaddr} Image; " \
 	"fatload mmc ${mmcdev} ${fdt_addr} ${fdt_file}; " \
 	"booti ${loadaddr} - ${fdt_addr};"
-- 
2.30.2

